### DISCUSSION TITLE
从1.9.9.4开始，我的ehv就无法使用了。

### DISCUSSION THREAD

**Discussion by muzikeyu** [xiaojieonly/Ehviewer_CN_SXJ#1698] [General]
打开后永远都显示“什么都没找到”，无论是搜索tag，还是打开收藏都一样显示“什么都没找到”。
设置里各种选项全部都试了个遍，要么没变化，要么显示“超时”。
刚才又装了1.9.9.6试了下，还是一样。
请教有没有解决方法？

**Comment by YamiNoHikari** [comment]
尝试退出登录，重启应用后通过网页登录。eh现在引入人机验证了，之前登录的记录可能会失效。

**Reply by muzikeyu** [reply]
问题解决，非常感谢。
